android/pytorch_android/src/main/cpp/pytorch_jni.cpp

```diff
@@ -418,7 +418,7 @@ class JIValue : public facebook::jni::JavaClass<JIValue> {
     } else if (JIValue::kTypeCodeLong == typeCode) {
       static const auto jMethodGetLong =
           JIValue::javaClassStatic()->getMethod<jlong()>("toLong");
-      return at::IValue{jMethodGetLong(jivalue)};
+      return at::IValue{(int64_t)jMethodGetLong(jivalue)};
```

Comment:
This is part of the Mac PR, right?

Reply:
yes, It is for mac, I will create separate pr for it.